hass_nabucasa/__init__.py

```diff
@@ -165,15 +175,11 @@ def run_executor(self, callback: Callable, *args) -> asyncio.Future:
         """Log a user in."""
         with async_timeout.timeout(10):
             await self.run_executor(self.auth.login, email, password)
-
-        self.run_task(self.iot.connect())
-
-        await self.client.logged_in()
+        await self.start()
```

Comment:
This is not good. Now we're going to call `async_initialize` twice on the client. We should make sure that we don't conflate starting up the cloud class and logging in .

Reply:
That is wrong on you PR on HA. We call initialize also on start, and that going now complete wrong. I try to fix this mistake now.

Reply:
I fix your comment now because on your PR you offload this part to an interface class function and that is not what a interface class function should do per design pattern

Reply:
In my PR, because the code was the same between getting logged in and initialization, I decided to just call that code and re-use it.